fetch.bs

```diff
@@ -6718,6 +6753,415 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+   <a for=/>top-level traversable</a> and <var>controlDocument</var> is not <a>allowed to use</a>
+   the <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch-minimal}}"
+   <dd>
+    <p>640 kibibytes
+    <p class="note allow-2119">640kb should be enough for everyone.
+
+   <dt><var>controlDocument</var>'s <a>node navigable</a> is a <a for=/>top-level traversable</a>
+   and  <a>allowed to use</a> the
+   <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch-minimal}}"
+   <dd>
+    <p>512 kibibytes
+    <p class=note>The default of 640 kibibytes, decremented By
+    <a>quota reserved for <code>deferred-fetch-minimal</code></a>)
+
+
+   <dt><var>controlDocument</var>'s <a>node navigable</a>'s <a>navigable container</a>'s
+   <a>reserved deferred-fetch quota</a> is <a for="reserved deferred-fetch quota">normal quota</a>,
+   and <var>controlDocument</var> is <a>allowed to use</a> the <a>policy-controlled feature</a>
+   "{{PermissionsPolicy/deferred-fetch}}"
+   <dd><a for="reserved deferred-fetch quota">normal quota</a>
+
+   <dt><var>controlDocument</var>'s <a>node navigable</a>'s <a>navigable container</a>'s
+   <a>reserved deferred-fetch quota</a> is <a for="reserved deferred-fetch quota">minimal quota</a>,
+   and <var>controlDocument</var> is <a>allowed to use</a> the <a>policy-controlled feature</a>
+   "{{PermissionsPolicy/deferred-fetch-minimal}}"
+   <dd><a for="reserved deferred-fetch quota">minimal quota</a>
+  </dl>
+
+ <li>
+  <p><a for=list>For each</a> <var>navigable</var> in <var>controlDocument</var>'s
+  <a>node navigable</a>'s <a>descendant navigables</a> whose <a>container document</a>'s
+  <a>deferred-fetch control document</a> is <var>controlDocument</var>, decrement <var>quota</var>
+  by <var>navigable</var>'s <a>navigable container</a>'s <a>reserved deferred-fetch quota</a>.
+
+  <p class=note>Delegate some of the quota to nested documents that reserved it.
+
+ <li><p>If <var>quota</var> is equal or less than 0, then return 0.
+
+ <li><p>Let <var>quotaForRequestOrigin</var> be 64 kibibytes.
+
+ <li>
+  <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
+  <var>controlDocument</var>'s <a>fetch group</a>'s
+  <a for="fetch group">deferred fetch records</a>:</p>
+
+  <ol>
+   <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+   <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+   <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+   <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+   <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+   then decrement <var>quotaForRequestOrigin</var> by <var>requestLength</var>.
+  </ol>
+
+ <li><p>If <var>quota</var> is equal or less than 0, then return 0.
+
+ <li><p>If <var>quota</var> is less than <var>quotaForRequestOrigin</var>, then return
```

Comment:
```
If quota is less than quotaForOrigin, then return quota.
```

(Discussed offline in chat, but copying here)

Given the following examples:

```
// root (a.com) -> frame-1 (a.com) -> frame-2 (b.com)
//              -> frame-3 (b.com) -> frame-4 (a.com)
```

I'd like to verify quotas for the above 5 documents:

```
root: 64kb? (the result I got, but isn't 8kb taken by frame-2?)
frame-1: 54kb
frame-2: 8kb
frame-3: 8kb
frame-4: 0kb?
```

The questionable part is `root` and `frame-1`, they can't get more than 64kb quota due to the quoted return step at the beginning.